src/gluonts/model/deepar/_estimator.py

```diff
@@ -140,6 +140,19 @@ class DeepAREstimator(GluonEstimator):
         The scaling coefficient of the temporal activation regularization
     batch_size
         The size of the batches to be used training and prediction.
+    minimum_scale
+        The minimum scale that is returned by the MeanScaler
+    default_scale
+        Default scale that is applied if the context length window is
+        completely unobserved. If not set, the scale in this case will be
+        the mean scale in the batch.
+    impute_missing_values
+        Whether to impute the missing values during training by using the
+        current model parameters. Recommended if the dataset contains many
+        missing values. However, this is a lot slower than the default mode.
+    imputation_samples
```

Comment:
Maybe we should change this name.
Currently by *imputation_samples* I understand that this variable contains samples.
What about *num_imputation_samples* or something alike?

Reply:
> Maybe we should change this name.
> Currently by _imputation_samples_ I understand that this variable contains samples.
> What about _num_imputation_samples_ or something alike?

Done